platform/iphone/gl_view.h

```diff
@@ -54,8 +54,8 @@
 	CADisplayLink *_displayLink;
 
 	// An animation timer that, when animation is started, will periodically call -drawView at the given rate.
-	// Only used if CADisplayLink is not
 	NSTimer *animationTimer;
+	// I'd like to remove the above as an option
```

Comment:
Hey thanks for chiming in as well, it's great to get feedback so quickly.

I hate to reply with just a link, but I think [this article](https://medium.com/@danielemargutti/the-secret-world-of-nstimer-708f508c9eb) does a fantastic job of laying out my concerns around `NSTimer`.

The TL;DR is basically this:

> Cocoa/UIKit defines several type of modes: on iOS there is a special mode called 

> UITrackingRunLoopMode : it set while tracking in controls takes place. It’s a fundamental piece because it allows UI events to be presented smoothly. For example in drag loop or other user interface tracking loops in this mode, processing in this mode will limit the input event. For example, when the finger UITableView drag will in this mode.
> While the main thread’s run loop is in UITrackingRunLoopMode , most background events like network callbacks, aren’t delivered. and no extra processing is done (and it means no lag during scrolling).
> 
> In this continuous loop to check events, NSRunLoop also check for timer interval elapsed event; once detected it will call method registered by the NSTimer class.
> 
> What it means for NSTimers ? It’s simple; usually they don’t fire when the user is scrolling a table or other UIScrollView — or doing anything else that puts the run loop in event tracking mode

Followed by

> This mean NSTimer cannot work as real-time timers: the point of the loop where fire events are managed might get delayed and you can loss precision.
> In fact both NSTimer and GCD timers are not suitable for realtime need (you cannot use them for latency-sensitive purposes, like video buffer sync or audio processing); the effective resolution of the time interval for a timer is limited to on the order of 50–100 milliseconds (hint: CADisplayLink).

At any rate, `CADisplayLink` is a much better alternative and VERY well supported on all devices.